requirements.txt

```diff
@@ -23,6 +23,8 @@ gunicorn==19.9.0
 psycopg2==2.7.7
 psycopg2-binary==2.7.7
 npm
+social-auth-app-django==3.1.0
+social-auth-core==3.2.0
 # REST API
```

Comment:
Since you added new packages in the requirements.txt. Include it also in your PR in the section of how should this be manually tested and notify us to install the requirements for this feature.
In addition. 
Before starting the server we have also to run migrations that come with the social-auth-app-Django.

Reply:
> In addition.
> Before starting the server we have also to run migrations that come with the social-auth-app-Django. Otherwise, if we don't include this, others will find problems with testing this feature.

@reifred I did not add this to the manual testing description initially because to test that these work we need to use the hosted application.